agent/jvm/pom.xml

```diff
@@ -68,6 +75,13 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
```

Comment:
I really wonder why this dependency would be necessary as the JVM agent doesnt use servlets at all (but the JVM internal HTTP server with its own custom API)